alerts/openshift-container-storage-operator/PersistentVolumeUsageNearFull.md

```diff
@@ -2,8 +2,9 @@
 
 ## Meaning
 
-Persistent Volume Claim (PVC) usage has exceeded 75% of its capacity,
+Persistent Volume Claim (PVC) usage is alarmingly very high,
 indicating an imminent risk of reaching full capacity.
+Please see the alert description for exact values.
```

Comment:
Why not use the same text explanation in all the runbooks?
_"Please see the alert documentation text for an exact threshold limit."_

Reply:
Done